app/templates/specialite.html

```
@@ -26,63 +26,74 @@ <h1 class="specialite-header__title">{{ specialite.nom }}</h1>
-      {% if interv.alternative_allergie %}
-      <div class="protocol-card protocol-card--allergie protocol-card--compact">
-        <div class="protocol-card__badge">Allergie</div>
-        {% for alt in interv.alternative_allergie %}
-        <div class="protocol-card__alt{% if not loop.first %} protocol-card__alt--separator{% endif %}">
-          <h4 class="protocol-card__molecule protocol-card__molecule--alt">
-            {{ alt.molecule }}
-            {% if alt.intention %}<span class="protocol-card__intention">({{ alt.intention }}{{ "ère" if alt.intention == 1 else "ème" }} intention)</span>{% endif %}
-          </h4>
+  {% for sous_cat, interventions in groupes.items() %}
+
+  {% if groupes | length > 1 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)